tests/plugins/toolsets/test_multi_instance.py

```
@@ -0,0 +1,228 @@
+                ],
+            }
+        )
+        r = _call(ts, {})
+        assert r.data["username"] == "u"
+        assert r.data["api_key"] is None  # atomic group dropped the global api_key
+
+
+class TestHealthAggregation:
+    def test_fails_only_when_all_unhealthy(self):
+        ts = multi_instance(_FakeToolset)
+        ok, msg = ts.prerequisites_callable(
```

Ruff (0.15.15)

[warning] Unpacked variable `msg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)